### PR TITLE
remove hard-coded pathname so test data populates

### PR DESCRIPTION
**Pull request by ex-nerd**
When starting this up with a test-data directory in my own code base, nothing is populated into the action.

It took a LOT of debugging to get this figured out (including an undocumented environment variable), but I think this will work properly now.

### CODE REVIEW

**Comment by fsouza** [comment]
omg sorry about that hahah

It seems that this is working now?

**Comment by ex-nerd** [comment]
Yeah, things should now work properly once this merges.

File: entrypoint.sh [hunk 1]
```diff
@@ -15,17 +15,31 @@ INPUT_PUBLIC_HOST=$(printenv INPUT_PUBLIC-HOST)
 docker_image=fsouza/fake-gcs-server:${INPUT_VERSION}
 
 if [ -n "${INPUT_DATA}" ]; then
-	if ! [ -d "${INPUT_DATA}" ]; then
+	if [ -n "${INPUT_DEBUG}" ]; then
+		echo "INPUT_DATA=${INPUT_DATA}"
+		echo "RUNNER_WORKSPACE=${RUNNER_WORKSPACE}"
+		echo "GITHUB_WORKSPACE=${GITHUB_WORKSPACE}"
+		echo "HOME=${HOME}"
+		env
```
Comment (fsouza) [inline entrypoint.sh:23]: Should we remove this call? Not sure if github will hide secrets that are environment variables? Don't want to leak people's secret if they're using this action x)

Comment (ex-nerd) [reply entrypoint.sh:23]: oops. yeah that is debug code left over.

Comment (ex-nerd) [reply entrypoint.sh]: And it's gone.

**Pull request merged**